aws-codeartifact-domain/aws-codeartifact-domain.json

```diff
@@ -41,7 +41,9 @@
     "/properties/EncryptionKey"
   ],
   "readOnlyProperties": [
-    "/properties/DomainOwner"
+    "/properties/DomainOwner",
+    "/properties/DomainName",
```

Comment:
[`readOnlyProperties`](https://github.com/aws-cloudformation/aws-cloudformation-resource-schema/#resource-semantics) cannot be specified by the customer, and will be removed from the [Resource Specification](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-resource-specification.html)
please track [here](https://i.amazon.com/ULURU-1033) and [here](https://github.com/aws-cloudformation/aws-cloudformation-coverage-roadmap/issues/68) for [`GetAtt`](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/intrinsic-function-reference-getatt.html) expansion beyond [`readOnlyProperties`](https://github.com/aws-cloudformation/aws-cloudformation-resource-schema/#resource-semantics)

```suggestion
```